setup_undercloud.yml

```diff
@@ -39,24 +38,6 @@
             hostname: "{{ hammer_host }}"
             ssh_user: "root"
 
-        - name: clone badfish
-          git:
-            repo: "https://github.com/redhat-performance/badfish.git"
-            dest: "{{ ansible_user_dir }}/badfish"
-
-        - name: Create badfish virtualenv
-          command: virtualenv {{ badfish_venv }} creates={{ badfish_venv }}
-
-        - name: Install badfish requirements
-          pip:
-            requirements: "{{ ansible_user_dir }}/badfish/requirements.txt"
-            virtualenv: "{{ badfish_venv }}"
-
-        - include_tasks: tasks/copykeys.yml
```

Comment:
Note: We are already adding keys at
https://github.com/redhat-performance/jetpack/blob/master/setup_undercloud.yml#L37
i.e this task is duplicated at L37 and L55.
So removing here i.e
https://github.com/redhat-performance/jetpack/blob/master/setup_undercloud.yml#L55